flips/README.md

```diff
@@ -0,0 +1,137 @@
+# Flow Improvement Proposals (FLIP)
+
+The purpose of a FLIP is to engage the Flow community in
+development by gathering feedback from contributors and experts while
+also communicating design changes broadly.
+
+## Who is involved?
+
+Any **community member** may help by providing feedback on whether the FLIP will
+meet their needs.
+
+A **FLIP author** is one or more community member who writes a FLIP and is
+committed to championing it through the  process.
+
+A **FLIP sponsor** is any maintainer who sponsors the FLIP and will shepherd it
+through the FLIP review process.
+
+A **review committee** is a group of maintainers who have the responsibility of
+recommending the adoption of the FLIP.
+
+## What is a FLIP?
+
+A FLIP is a document that describes a requirement and the proposed changes that
+will solve it. Specifically, the FLIP will:
+
+* be formatted according to the FLIP template
+* be submitted as a pull request to the
+  [onflow/flow/flips](https://github.com/onflow/flow/tree/master/flips) directory
+* be subject to discussion and a review meeting prior to acceptance
+
+## FLIP process
+
+Before submitting a FLIP, it is a good idea to discuss your aims with project
+contributors and maintainers and get early feedback. Use the [Flow community forum](https://forum.onflow.org/) or [Discord server](https://discord.gg/flow). After writing the FLIP draft, get feedback from these experts before submitting it.
+
+1. Recruit a sponsor from the maintainers of the project which your FLIP concerns.
+
+   Identify them in the FLIP, before posting the PR in step 2.
+   If no sponsor is found you may still post the FLIP, but if 
+   within a month of posting the PR there is still no sponsor,
+   it will be closed.
+
+2. Submit your FLIP as a pull request to [onflow/flow](https://github.com/onflow/flow). 
+
+   Name your FLIP file using the [template](./yyyymmdd-flip-template.md) `YYYYMMDD-descriptive-name.md`, where
+   YYYYMMDD is the date of submission, and ‘descriptive-name’ relates to the
+   title of your FLIP. For instance, if your FLIP is titled “Event Streaming API”,
+   you might use the filename `20180531-event-streaming-api.md`. If you have images
+   or other auxiliary files, create a directory of the form `YYYYMMDD-descriptive-name`
+   in which to store those files.
+
+   Include the header table and the contents of the **Objective** section 
+   in the comment of your pull request, using Markdown. For an example,
+   please see [this example FLIP](https://github.com/onflow/flow/pull/5). Include a mention of any of the GitHub handles of co-authors, reviewers, and sponsors.
+
```

Comment:
Maybe mention that the author should attempt to fill as much as possible. 
Given that there are many and they don't all necessary apply, authors might feel "blocked" if they can't fill them. 

Maybe we can even categorize the sections in the template to be required or optional?